app/views.py

```diff
@@ -273,3 +273,158 @@ def admin():
         return "You cannot access this page.", 401
 
     return render_template('admin.html')
+
+
+@app.route('/users/create', methods=['GET', 'POST'])
+def create_user():
+    # Check if logged in first
+    if not is_logged_in():
+        if request.method == 'GET':
+            return redirect(url_for('login', next=request.url_rule))
+        elif request.method == 'POST':
+            return "Not logged in.", 401
+
+    form = forms.CreateUserForm()
+
+    ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
+                           test=app.config['TESTING'],
+                           token=session['token'])
+
+    # If form submitted (POST)
+    if form.validate_on_submit():
+        username = form.username.data
+        password = form.password.data
+        display_name = form.display_name.data
+        email = form.email.data
+        site_admin = form.site_admin.data
+        site_spectator = form.site_spectator.data
+        site_manager = form.site_manager.data
+        active = form.active.data
+
+        user = {
+            "username": username,
+            "password": password
+        }
+
+        if display_name:
+            user["display_name"] = display_name
+        if email:
+            user["email"] = email
+        if site_admin:
+            user["site_admin"] = site_admin
+        if site_spectator:
+            user["site_spectator"] = site_admin
+        if site_manager:
+            user["site_manager"] = site_admin
+        if active:
+            user["active"] = active
+
+        res = ts.create_user(user=user)
+
+        # TODO: Better error handling
+        if 'error' in res:
+            flash("Error: " + res["error"] + "\nText: " + res["text"])
+        elif 'pymesync error' in res:
+            flash("Error: " + res["pymesync error"] + "\nText: " +
+                  res["text"])
+
+        flash("Time successfully submitted.")
+        return redirect(url_for('create_user'))
+
+    # Flash any form errors
+    for field, errors in form.errors.items():
+        for error in errors:
+            flash("%s %s" % (
```

Comment:
The % string formatting is being depricated in python3, and it's good practice to use str.format() instead - please replace this with .format syntax:

`"{0} {1}".format(getattr(form, field).label.text, error)`